crates/core/tedge_api/src/alarm.rs

```diff
@@ -42,6 +42,13 @@ pub enum ThinEdgeJsonDeserializerError {
 
     #[error("Unsupported external device ID in topic: {0}")]
     UnsupportedExternalDeviceId(String),
+
+    #[error("Deserialing of alarm message: {payload}, received on topic: {topic}, failed due to error: {error}")]
```

Comment:
```suggestion
    #[error("Parsing of alarm message received on topic: {topic} failed due to error: {error}. Snipped payload: {payload}")]
```

Moving the payload to the end to make the output better.

Reply:
Make sense, fixed.